lib/rouge/lexers/hocon.rb

```diff
@@ -51,9 +77,9 @@ class HOCON < JSON
         rule %r/"""/, Literal::String::Double, :pop!
       end
 
-      state :interpolation do
-        rule %r/[\w\-\.]+?/, Name::Variable
-        rule %r/}/, Literal::String::Interpol, :pop!
+      prepend :constants do
+        # Numbers (handle the case where we have multiple periods, ie. IP addresses)
+        rule %r/\d+\.(\d+\.?){3,}/, Literal #
```

Comment:
The trailing octothorp seems unnecessary.
```suggestion
        rule %r/\d+\.(\d+\.?){3,}/, Literal
```

Reply:
@ashmaroli Cheers! That was indeed a typo.

Reply:
Hmmm, won't let me apply the suggestion for some reason. I'll make the change and update a bit later.

Reply:
Probably because I don't have write access to the branch. (Only maintainers and the author have that privilege)

Reply:
That sort of makes sense, I guess. Although if I'm applying the suggestion and I have access, you'd think it would work.